pgjdbc/build.gradle.kts

```diff
@@ -71,15 +71,15 @@ val String.v: String get() = rootProject.extra["$this.version"] as String
 
 dependencies {
     constraints {
-        api("com.github.waffle:waffle-jna:1.9.1")
-        api("org.osgi:org.osgi.core:6.0.0")
-        api("org.osgi:org.osgi.service.jdbc:1.0.0")
+        api("com.github.waffle:waffle-jna:3.3.0")
```

Comment:
Is this change required for module?

Reply:
As-is, the dependencies don't have automatic module names, which causes compilation failure. The versions I bumped to do.

Reply:
I could downgrade and remove the static requirements
```
  requires static com.sun.jna;
  requires static com.sun.jna.platform;
  requires static waffle.jna;
  requires static org.osgi.service.jdbc;
  requires static osgi.core;
```
from the module, but then if those classes (like PGBundleActivator) are loaded you'd see an exception
```
Exception in thread "main" java.lang.IllegalAccessError: superinterface check failed: class org.postgresql.osgi.PGBundleActivator (in module org.postgresql.jdbc) cannot access class org.osgi.framework.BundleActivator (in unnamed module @0x4fca772d) because module org.postgresql.jdbc does not read unnamed module @0x4fca772d
```

Reply:
We could leave the requirements out and just mark waffle and osgi as non-required under modules. Not sure how that would affect future modular versions of waffle or osgi.